markdown/assignments/pool_a/speichern_und_laden/index.md

```diff
@@ -7,9 +7,15 @@ weight: 5
 hidden: true
 ---
 
+## Ziel
+
+In dieser Aufgaben sollen Sie ein System zum Speichern und Laden implementieren.
 
 ## Speichern und Laden
 
-Implementieren Sie eine Möglichkeit um das Spiel und den aktuellen Zustand zu speichern und zu laden.
-Überlegen Sie sich, was alles gespeichert werden muss und wie Sie die Daten dafür ablegen müssen.
-Speichern Sie mindestens den aktuellen Zustand des Helden und das aktuelle Level sowie alle Elemente die Sich im Level befinden (und wo).
+Implementieren Sie eine Möglichkeit, um das Spiel und den aktuellen Zustand zu speichern und zu laden.
+Überlegen Sie sich, was alles gespeichert werden muss und wie Sie die Daten dafür ablegen müssen. 
+Speichern Sie mindestens den aktuellen Zustand des Helden und das aktuelle Level sowie alle Elemente, die sich im Level befinden (und wo).
```

Comment:
Ist das klar genug? Was genau soll vom aktuellen Level gespeichert werden? Ich glaube, es fehlt ein Zusatz in der Art "Speichern Sie die Informationen über den Spielzustand, damit Sie zu einem späteren Zeitpunkt ein neues Spiel starten und den jetzigen Spielzustand durch Laden der abgespeicherten Daten wiederherstellen können."

Reply:
Und dürften die Studis einfach serialisieren? Das geht ja im Zweifel rekursiv, d.h. man muss einfach nur den richtigen Klassen die richtige Ableitung verpassen und fertig.

Aber das würde auch Klassen aus der Vorgabe einschließen ...

Reply:
> Und dürften die Studis einfach serialisieren? Das geht ja im Zweifel rekursiv, d.h. man muss einfach nur den richtigen Klassen die richtige Ableitung verpassen und fertig.

Ich bin nicht sicher worauf du hinaus willst. Ist mein Kaffee zu schwach?